### PR TITLE
remove torquebox proxy

### PR DESCRIPTION
**Pull request by eiswind**
this is a working example that replaces the torquebox proxy with the gem-wagon extension

### CODE REVIEW

**Comment by michaelmollioy** [comment]
test

File: asciidoc-to-revealjs-example/pom.xml [hunk 1]
```diff
@@ -47,6 +47,13 @@
 
     <build>
         <defaultGoal>process-resources</defaultGoal>
+        <extensions>
+            <extension> <!-- this allows us to download gems -->
+                <groupId>org.torquebox.mojo</groupId>
+                <artifactId>mavengem-wagon</artifactId>
+                <version>1.0.3</version>
+            </extension>
```
Comment (mtcmolloy) [inline asciidoc-to-revealjs-example/pom.xml]: no comment

**Comment by abelsromero** [comment]
I tried without the extension and last version of `gem-maven-plugin` (1.1.8), and the example also work. Does the same happen to you?

**Comment by eiswind** [comment]
I tried the following:
Deleted rubgems/asciidoctor-revealjs from my .m2/repository
Removed the extension from the pom.
updated to 1.1.8

did not work

**Comment by abelsromero** [comment]
haha, I was just about to post that I had found it failed in a clean machine, but hadn't found the default cache location yet. thanks again.

In that case, if you feel like going for an extra mile it'd be nice to update the versions of the plugins (download won't work I recall though, 1.3.0 is fine) and gems. If not, I can merge and review them later.

**Comment by fredmoal** [comment]
Nice !
works fine with new asciidoctor-revealjs 3.0.0
thanks !!

**Comment by eiswind** [comment]
Did not look at that :) Just updated plugins and gem.

**Comment by fredmoal** [comment]
I've got some trouble with upgrade to download-maven-plugin 1.4.2 (ClassNotFoundException)
solution : delete the directory .m2/repository/.cache/download-maven-plugin/
seems to work now with 1.4.2

[ERROR] Failed to execute goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.2:wget (install-revealjs) on project cours: Execution install-revealjs of goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.2:wget failed: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.2:wget (install-revealjs) on project cours: Execution install-revealjs of goal com.googlecode.maven-download-plugin:download-maven-plugin:1.4.2:wget failed: java.lang.ClassNotFoundException: com.googlecode.download.maven.plugin.internal.CachedFileEntry

**Comment by eiswind** [comment]
Didn't notice this, because I smartly deleted it in advance ...

**Comment by abelsromero** [comment]
> I've got some trouble with upgrade to download-maven-plugin 1.4.2 (ClassNotFoundException)
> solution : delete the directory .m2/repository/.cache/download-maven-plugin/
> seems to work now with 1.4.2

Thanks for the tip. I was quite misslead by the error message and never though of it :smile:

**Pull request merged**